js/player.js

```diff
@@ -138,7 +138,14 @@ Flickity.prototype._createPlayer = function() {
 
   this.on( 'activate', this.activatePlayer );
   this.on( 'uiChange', this.stopPlayer );
-  this.on( 'pointerDown', this.stopPlayer );
+
+  if ( this.options.stopOnClick ) {
+    this.on( 'pointerDown', this.stopPlayer );
```

Comment:
I would change the name of the option to `stopOnPointerDown`.

Rather than point to stop/pause/unpause methods, I think this logic should be handled in a generic `onPointerDownPlayer` method, where the `stopOnClick` logic is.